docs/users/welcome.md

```diff
@@ -22,3 +22,35 @@ What to do (and not to do) in the community.
 - Yuck Someone Else's Yum, as they say.
   - If you have serious concerns, do bring them up, either privately or on the appropriate Board. Please remember the "do" rules, and be mindful that the present fandom climate might make some people twitchier about certain subjects.
 - Continue pressing an issue after someone has asked you to stop. In threads, **OP has final saying on what is allowed on their thread**. Making a different thread to discuss something OP would rather not touch upon is allowed, but try to avoid "subtle" jabs at the previous OP (trust us, they're never subtle).
+
+## FAQS
+
+### Can I necropost*?
+
+On BobaBoard, you can necro everything you want.
+
+*a.k.a. answer to older threads that haven't seen activity in a while
+
+### I really need a post deleted!
+
+Just write under the post that you'd like for that to happen, and I'll do so after confirmation. If I don't notice the request, please bump it up in !bobaland, which I check most often. 
+
+If you see a deletion request in the wild, feel free to *privately* bring it up to me on Discord. You don't need to tell me if it's yours 😉
+
+### Is BobaBoard a [Insert Social Network] Clone?
```

Comment:
I would rephrase this in a way that helps clarify expectations of what we have now as software vs what people can expect to see going forward. This FAQ makes sense in the abstract sense, but I'd keep the welcome guide more tightly focused on the practical. We can always link out as needed.

Reply:
Post deletions & necroposting were both questions users have asked about in terms of boba etiquette & that didn't quite seem appropriate for the 'main' etiquette section. I was also planning on putting some sort of 'how do i know what board to post on'? section in here since that was one of the more popular requests on this post: https://v0.boba.social/!bobaland/thread/45d78a36-f465-428e-a1d4-c1fdae6e9e52

The 'is bobaboard a [x] clone? is more general though and i agree that could prob be cut/moved somewhere else

Reply:
Actually correction: the "can i necropost" is currently unchanged from the orig bobadoc -- am not sure why that section got clipped off when it made the jump to github but if that was intentional i don't mind moving the FAQ?